### PR TITLE
Wire up Ticket registration count | Closes #2656

### LINKED ISSUE #2656: Complete Tickets registration count todo
`registrationCount` needs  to added to Tickets table view

### PR DESCRIPTION
**Pull request by manzoorwanijk**
This PR adds `registrationCount` to `Ticket` object in GQL schema and updates the consumer components to use it.

### CODE REVIEW

File: assets/src/domain/eventEditor/ui/tickets/hooks/useTicketsRegistrationCount.ts [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-const useTicketsRegistrationCount = () => {
-	// @todo get registration count
-	const allTicketsRegistrationsCount = {}; // {ticket.id: RegCount}
-
-	const getRegCount = (ticketId: EntityId): number => {
-		return allTicketsRegistrationsCount[ticketId] || 0;
-	};
-
-	return { getRegCount };
-};
```
Comment (nicolad) [inline assets/src/domain/eventEditor/ui/tickets/hooks/useTicketsRegistrationCount.ts:15]: nice that you removed this, it was a bit confusing

Comment (manzoorwanijk) [reply assets/src/domain/eventEditor/ui/tickets/hooks/useTicketsRegistrationCount.ts:15]: Agreed

File: assets/src/domain/eventEditor/services/apollo/mutations/tickets/useOptimisticResponse.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ export const TICKET_DEFAULTS: Ticket = {
 	price: null,
 	prices: null,
 	quantity: -1,
+	registrationCount: 0,
```
Comment (tn3rb) [inline assets/src/domain/eventEditor/services/apollo/mutations/tickets/useOptimisticResponse.ts:35]: for brevity we _could_ rename this to `regCount` but that's not a necessity

Comment (manzoorwanijk) [reply assets/src/domain/eventEditor/services/apollo/mutations/tickets/useOptimisticResponse.ts:35]: First I did that, then I thought this one is more clear 😄

**Pull request merged**